briskLimbs/briskLimbs
# [feature/BRIS-4] video uploads

https://brisklimbs.atlassian.net/browse/BRIS-4

## What it solves
It provides video uploads functionality with proper validations.

## How this PR fixes it
Some of the things included in this PR are:

1. File, video and video validation services
2. Video table and model
3. Settings table and model with seeder
4. Introduces jQuery
5. Upload page powered by Dropzone
6. Upload page validations in FE and BE
7. Helper functions

## How to test it
1. Go to http://localhost/videos/upload
2. Try to upload a video
3. You should get size errors if you exceed limits
4. You should get extension errors if you try something unexpected
5. Video should upload if no errors
6. Video should update using post-upload form

## Tested browsers


## Readiness Checklist

### Author/Contributor
- [ ] If documentation is needed for this change, has that been included in this pull request
- [x] Pull request title is brief and descriptive (for a changelog entry)

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [x] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`
